src/reporting/interfaces/ExtractionProcess.ts

```diff
@@ -203,6 +223,54 @@ export interface ExtractionStatus {
   _links: ExtractionStatusLinks;
 }
 
+/**
+ * List of Extractions.
+ * @export
+ * @interface ExtractionCollection
+ */
+export interface ExtractionCollection {
+  /**
+   * List of Extraction Logs.
+   * @type {Array<ExtractionLog>}
+   * @memberof ExtractionCollection
+   */
+  extractions: Array<Extraction>;
+  /**
+   *
+   * @type {PagedResponseLinks}
+   * @memberof ExtractionCollection
+   */
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  _links: PagedResponseLinks;
+}
+
+/**
+ * Extraction Status properties.
+ * @export
+ * @interface Extraction
+ */
+export interface Extraction {
+  /**
+   * Unique Identifier for the Extraction Run. Use this to check run status.
+   * @type {string}
+   * @memberof Extraction
+   */
+  jobId: ExtractorState;
+  /**
+   * Additional justification for the current state of the Extraction Run.
```

Comment:
`startedOn` is not a justification.

Reply:
Fixed comments (forgot about them before creating the PR, my bad).